TrainingJava/src/ShoppingCartDiscount/Product.java

```diff
@@ -0,0 +1,54 @@
+    /*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package ShoppingCartDiscount;
+
+/**
+ *
+ * @author andreslietti
+ */
+   // Class Product with attributes and constructors.
+ public class Product {
+     
+     private String name;
+     private float price;
+     private int quantity;
```

Comment:
This is good for now, but it seems a bad design, since you have a product, and quantity is not an infered field for the class product. You can add the quantity in other model class like Sell, Shop, or something like that and add a List<Product> in that class. But here you are  making a hard-link between quantity and the product.